app/templates/frameworks/edit_declaration_section.html

```diff
@@ -59,29 +60,41 @@
             {% endfor %}
 
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-            {% if is_last_page %}
+
+            {% if next_section %}
+              {# If there is a next page show the 'continue' button' #}
               {%
                 with
-                label="Make declaration",
+                label="Save and continue",
                 type="save"
               %}
                 {% include "toolkit/button.html" %}
               {% endwith %}
-              <p class="last-edited move-to-complete-hint">
-                You can edit the declaration until the application deadline.
+
+              {# If there is a next page show the 'next page' info #}
+              <p class="next-page-message">
+                Next: {{ next_section.name }}
               </p>
 
+              {# If there is a next page the 'save and return' button is secondary #}
+              {% with
+                type = "secondary",
+                label = "Save and return to declaration overview",
+                name = 'save_and_return_to_overview'
+              %}
+                {% include "toolkit/button.html" %}
+              {% endwith %}
+
             {% else %}
-              {%
-                with
-                label="Save and continue",
-                type="save"
+              {# If there is no next page the 'save and return' button is the primary and only save button #}
+              {% with
+                type="save",
```

Comment:
Does jinja allow `... if ... else ...` statements here? Because save<->secondary seems to be the only difference between these two `toolkit/button.html` uses...